uo_sys_framework/ACREModule/cfgXEH/init/ClientPreInit.sqf

```diff
@@ -0,0 +1,410 @@
+/*	Description: Gives actions to vehicles with AI Driver enabled
+ *	Arguments:
+ * 		N/A
+ *	Return Value:
+ * 		ARRAY
+ *	Author
+ *		PiZZADOX
+ */
+
+
+#define COMPONENT ACRE
+#include "\x\UO_FW\addons\Main\script_macros.hpp"
+UO_FW_EXEC_CHECK(CLIENT);
+
+["UO_FW_SettingsLoaded",
+{
+	if !(UO_FW_Server_AcreModule_Allowed) exitwith {};
+	if !(missionNamespace getvariable ["UO_FW_ACRE_SETTINGS_Enabled",false]) exitwith {};
+
+	if (!isDedicated && hasinterface) then
+	{
+
+		["UO_FW_RegisterModuleEvent", ["ACRE Setup", "Module for Acre Settings", "PiZZADOX and Sacher"]] call CBA_fnc_globalEvent;
+
+		UO_FW_Presets = ["default2", "default3", "default4", "default"];
```

Comment:
A lot of these can be made into private local vars now that they are all in the same event